department-of-veterans-affairs/abd-vro
# Bip Api using Xample as the model

## What was the problem?
BIP didn't have an API

Associated tickets or Slack threads:
- [1594](https://github.com/department-of-veterans-affairs/abd-vro/issues/1594)

## How does this fix it?[^1]
used the RMQ interface like in svc-xample-j to invoke already existing code dealing with BIP 

## How to test this PR
At this pint the tests are not guaranteed to work.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
